.github/workflows/publish-docs-and-website.yaml

```diff
@@ -44,16 +44,15 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
-      - name: Build library, documentation and website
+      - name: Build workspace packages and documentation
         run: |
           pnpm build
           pnpm build:docs
-          pnpm build:website
```

Comment:
Not immediately obvious to me why we're removing this step?

Reply:
Running `pnpm build` in the root builds all workspaces, `website` and `packages/catppuccin-unocss`. Since `build:docs` is not its own workspace and part of `packages/catppuccin-unocss`, we need to run it separately.